contributions/catalog/requirements.txt

```diff
@@ -1,4 +1,5 @@
 Flask==1.1.1
 pymongo==3.7.2
```

Comment:
You probably want to add Flask-Cors, if you want to add CORS in catalog

Reply:
I tested and it looks like catalog does not need CORS now ...so i will remove flask-cors related stuff

Reply:
If so, you might want to remove all the cors related codes from catalog